dev/strongpoc/models.py

```diff
@@ -3,61 +3,67 @@
 from django.db import models
 
 
-class BaseModel(models.Model):
-    def prep_dict(self, expand=None, self_name=None):
-        if isinstance(expand, basestring):
-            expand = [expand]
-        if expand is None:
-            expand = []
-
-        if self_name in expand:
-            expand.remove(self_name)
-
-        return {
-            "id": self.id
-        }
+def prep_expands(expand, self_name):
```

Comment:
This "common function across subclasses" is the perfect use case for an instance method. I support keeping it as such unless you can justify otherwise. 

I'm open.